lib/pages/laws.dart

```diff
@@ -20,7 +20,7 @@ class Laws extends StatelessWidget {
           crossAxisAlignment: CrossAxisAlignment.center,
```

Comment:
I would wrap all of this, everything that is inside the Column widget in a ClipRect, as right now there is pixel overflow from the list.